contracts/Governance/VestingToken.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.5;
+
+/// Token vesting contract.
+/// Adapted from Melonport AG <team@melonport.com>
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VestingToken {
+    using SafeMath for uint256;
+
+    ERC20 public token; // Radicle ERC20 contract
+    address public owner; // deployer; can interrupt vesting
+
+    bool public interrupted; // whether vesting is still possible
+    bool public isVestingStarted; // whether vesting period has begun
+    uint256 public totalVestingAmount; // quantity of vested token in total
+    uint256 public vestingStartTime; // timestamp when vesting is set
+    uint256 public vestingPeriod; // total vesting period in seconds
+    address public beneficiary; // address of the beneficiary
+    uint256 public withdrawn; // quantity of token withdrawn so far
```

Comment:
If we reversed the logic and stored `unwithdrawn`, we'd elegantly zero the storage on the end of life of the contract. Probably not that important :D

Reply:
Ahahaha maybe something we can do afterwards if we have extra time!